_posts/2025-07-01-first-year-at-wcc.html

```diff
@@ -0,0 +1,215 @@
+blurb_img: /assets/images/blog/2025-07-01-first-year-at-wcc.jpg
+blurb_img_source:
+description: "Our first year as WomenCodingCommunity"
+category: Community
+---
+
+<div class="text-justify">
+    <p>
+    Standing on the rooftop of Encode Hub on May 29th, looking out at faces of women who refused to let something meaningful disappear, I felt this overwhelming sense of "we actually did it."
+    </p>
+
+    <p>
+    I'm Rajani Rao, co-founder and director of the Women Coding Community. That night, as I looked out at everyone gathered for our first anniversary celebration, all I could think about was how absolutely surreal this whole journey has been.
+    </p>
+
+    <h2>When the World Said "No," We Said "Not Today"</h2>
+
+    <p>
+    You remember the moment. We all do. The diversity initiatives getting axed. Funding evaporating overnight. Women Who Code shutting down. It felt like watching something you cared about just... die.
+    </p>
+
+    <p>
+    But instead of accepting that ending, we chose to begin again.
+    </p>
+
+    <p>
+    We didn't wait around for some savior organization to emerge. We didn't sit there hoping someone else would fix it. We showed up and said, "Fine. We'll do it ourselves."
+    </p>
+
+    <p>
+    That's literally how the Women Coding Community was born - out of pure stubborn refusal to let something important disappear. And when we needed support to make it real, partners like SurrealDB stepped up and believed in our vision.
+    </p>
+
+    <h2>Building From Scratch While Juggling Everything</h2>
+
+    <p>
+    The energy that night was... honestly, it was everything I hoped it would be. One woman came up to me afterward and said, "I had such a great time! Really appreciated how welcoming and friendly everyone was, loved the energy!"
+    </p>
+
+    <p>
+    And I'm thinking, "Yes. THIS. This is exactly what we were trying to create."
+    </p>
+
+    <p>
+    Workshops, mentorship, study groups, career clubs and more -we've created all this from scratch. And we did it while holding jobs, raising families, learning new tech, and sometimes just surviving.
+    </p>
+
+    <p>
+    But that's what we do, isn't it? We build. We uplift. We persist. Even when we're running on three hours of sleep and our imposter syndrome is screaming.
+    </p>
+
+    <p>
+    The growth has been steady and real. Here are some highlights:
+    <ul>
+        <li>Our mentorship program achieved an 85% match success rate</li>
+        <li>Over 500 women joined our ML track</li>
+        <li>Nearly 180 women signed up for our new LeetCode track before we even launched it</li>
+    </ul>
+    But the numbers don't tell the real story. The real story is what one student told me that night: "Being surrounded by so many inspiring women reminded me that I can do this. That we all can. Sometimes, all it takes is seeing people who look like you doing what you dream of and suddenly, it feels possible."
```

Comment:
```suggestion
    But the numbers don't tell the real story. The real story is what one student told me that night: 
        <div class="container">
        <blockquote class="blockquote text-center">
          <p>Being surrounded by so many inspiring women reminded me that I can do this. That we all can. Sometimes, all it takes is seeing people who look like you doing what you dream of and suddenly, it feels possible.</p>
          <footer class="blockquote-footer">Anonymous student</footer>
        </blockquote>
    </div>
```

I believe it’s worth highlighting.